libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -18,6 +18,17 @@ const AP_Param::GroupInfo AC_AttitudeControl::var_info[] = {
 
     // 3 was for ACCEL_RP_MAX
 
+#if APM_BUILD_TYPE(APM_BUILD_ArduSub)
+    // @Param: ACCEL_Y_MAX
```

Comment:
It was needed to change the default value. I added a define instead of changing Copter's define.